### PR TITLE
Fixes #6756 - exposes vmware resource pools and folders through the API

### PR DESCRIPTION
**Pull request by gpetras**
This lets users get a list of resource pools and folders through the v2 API.

### CODE REVIEW

**Comment by dLobatog** [comment]
[test]

File: app/controllers/api/v2/compute_resources_controller.rb [hunk 1]
```diff
@@ -76,6 +77,13 @@ def available_clusters
         render :available_clusters, :layout => 'api/v2/layouts/index_layout'
       end
 
+      api :GET, "/compute_resources/:id/available_folders", "List available folders for a compute resource"
```
Comment (domcleal) [inline app/controllers/api/v2/compute_resources_controller.rb:80]: We've had a change to API docs for i18n support, please wrap the descriptions in `N_(..)` so they're extracted

**Comment by domcleal** [comment]
Works well, thanks.  If you can fix the API doc issue & rebase, I think it's good.

**Comment by domcleal** [comment]
Merged as fe1a273 for Foreman 1.7.0, thanks @gpetras!

**Pull request closed without merging**